web-client/instructions/Release note for Android.md

```diff
@@ -0,0 +1,48 @@
+# Release note for Android
+
+## Prerequisite
+* Android studio
+* Android SDK
+* Java
+
+## [Configuration](https://capacitorjs.com/docs/android/configuration#configuring-androidmanifestxml)
+
+Android apps manage permissions, device features, and other settings in the `AndroidManifest.xml` file, which is located at `android/app/src/main/AndroidManifest.xml`.
+
+### [Changing the Package ID](https://capacitorjs.com/docs/android/configuration#changing-the-package-id)
+
+To change your app’s Package ID (aka Application ID for Android), edit `applicationId` at the top of `android/app/build.gradle`
+
+> **Note**: you can also edit the version name & version code in the same file
+
+
+## [Open Android Studio](https://capacitorjs.com/docs/android#opening-the-android-project)
+
+To open the project in Android Studio, run:
+
+`$ ionic capacitor open android`
+
+> Alternatively, you can open Android Studio and import the android/ directory as an Android Studio project.
+
+
+[![Android studio](https://i.postimg.cc/jShx1V5V/Screenshot-2022-01-27-at-17-40-34.png)](https://postimg.cc/yDkCJbcn)
```

Comment:
It might be better to host these screenshots in the repo, or at least the repo wiki, rather than linking to postimg.cc?